src/System.Memory/src/System/ThrowHelper.cs

```diff
@@ -106,17 +106,15 @@ public static bool TryParseThrowFormatException<T>(out T value, out int bytesCon
         }
     }
 
+    //
+    // The convention for this enum is using the argument name as the enum name
```

Comment:
VS complains that the enum names are not capitalized and hence not following the coding guideline. I have added this comment here from coreclr. Otherwise we would have to make the changes to the enum there and the effort may not be worth it for this case.

https://github.com/dotnet/coreclr/blob/master/src/mscorlib/src/System/ThrowHelper.cs#L395

Let me know if anyone disagrees and I will fix the name violation:
![image](https://user-images.githubusercontent.com/6527137/35762228-2335ef28-0848-11e8-9a31-8cbb9004d61d.png)

Reply:
I don't think it's worth "fixing".  This is a special case.